cnv/cnv_users_guide/cnv-importing-virtual-machine-images-datavolumes.adoc

```diff
@@ -0,0 +1,28 @@
+[id="cnv-importing-virtual-machine-images-datavolumes"]
+= Importing virtual machine images with DataVolumes
+include::modules/cnv-document-attributes.adoc[]
+:context: cnv-importing-virtual-machine-images-datavolumes
+toc::[]
+
+You can import an existing virtual machine image into an object in your
```

Comment:
into an object is confusing.  Is it better to say into your openshift cluster or into your storage or "import an existing vm image for use with CNV"?

Reply:
Agreed, I wasn't sure about that. I'm changing it to "into your OpenShift Container Platform cluster" because the next sentence starts with "Container-native Virtualization." Thanks.